nameable.rb

```diff
@@ -0,0 +1,5 @@
+class Nameable
+  def correct_name
+    'NotImplementedError'
+  end
```

Comment:
- [ Required ] I think it would be nice if this method actually `raised` an `NotImplementedError` instead of just returning a string. I would like to suggest that you refactor this code as shown below so that it raises an error when a class that inherits from `Nameable` does not implement the `correct_name` method.

## Example code
```ruby
class Nameable
  def correct_name
     raise NotImplementedError, "#{self.class} has not implemented method '#{__method__}'"
  end
end
```

## Project requirements
![Screenshot from 2023-09-01 00-21-50](https://github.com/sergio-usma/ruby_school_library/assets/16392046/c11304fc-a327-4ca4-aca6-3f6652aa4c71)